go/libkb/env.go

```diff
@@ -455,6 +457,20 @@ func (e *Env) GetSocketFile() (ret string, err error) {
 	return
 }
 
+func (e *Env) GetGregorURI() string {
+	return e.GetString(
+		func() string { return os.Getenv("GREGOR_URI") },
+		func() string { return e.config.GetGregorURI() },
```

Comment:
we probably need a default fallback here.  Plus, it's not really a URI if it's just a hostname. So either: (1) we should make it a URI ("tcp://gregor.api.keybase.io:443" or whatever) or rename it to a hostname.  And we should potentially allow ports other than 443, but we should keep it on 443 for firewalls.